sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -73,12 +73,13 @@
     from azure.core.credentials import AzureNamedKeyCredential, AzureSasCredential, TokenCredential
     from datetime import datetime
     from ._generated.models import BlockList
-    from ._models import (
+    from ._models import (  # pylint: disable=unused-import
```

Comment:
Why add the import is its unused? :) (Applies to many places in this review)

Reply:
I believe I put this here because we `BlobTokenAudience` is showing up in docstrings but technically we don't use it anywhere in code (so it was greyed out as unused import). I am actually unsure of the behavior if you don't import but use it in docstring, so can plop it off and see on one of these files!